src/nbl/asset/IAssetManager.cpp

```diff
@@ -307,9 +307,9 @@ void IAssetManager::insertBuiltinAssets()
 	// samplers
 	{
 		asset::ISampler::SParams params;
-		params.TextureWrapU = asset::ISampler::ETC_REPEAT;
-		params.TextureWrapV = asset::ISampler::ETC_REPEAT;
-		params.TextureWrapW = asset::ISampler::ETC_REPEAT;
+		params.TextureWrapU = asset::ISampler::E_TEXTURE_CLAMP::ETC_REPEAT;
+        params.TextureWrapV = asset::ISampler::E_TEXTURE_CLAMP::ETC_REPEAT;
+        params.TextureWrapW = asset::ISampler::E_TEXTURE_CLAMP::ETC_REPEAT;
```

Comment:
indentation got messed up

also at the time you have to type out `asset::ISampler::E_TEXTURE_CLAMP` I'd use a local `using` alias on line 309